net-libs/luvi/luvi-2.15.0.recipe

```diff
@@ -0,0 +1,57 @@
+SUMMARY="A project in-between luv and luvit"
+DESCRIPTION="luvi is a project in-between luv and luvit.
+The goal of this is to make building luvit and derivatives much easier."
+HOMEPAGE="https://github.com/luvit/luvi"
+COPYRIGHT="2012 The Luvit Authors
+	Portions of this software were based on Node.js, \
+	Joyent, Inc. and other Node contributors.
+	"
+LICENSE="Apache v2"
+REVISION="1"
+SOURCE_URI="https://github.com/luvit/luvi/releases/download/v$portVersion/luvi-source.tar.gz"
+CHECKSUM_SHA256="91f40fb6421888c2ee403de80248250c234f3bfb6dd1edbbc9188a89e4b9708a"
+SOURCE_FILENAME="luvi-$portVersion.tar.gz"
+SOURCE_DIR=""
+
+ARCHITECTURES="all !x86_gcc2"
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+SECONDARY_ARCHITECTURES="?x86" # luv has this, and official Windows build of luvi supports 32 bit, so assuming that it works
+fi
+
+PROVIDES="
+	luvi = $portVersion
+	cmd:luvi
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
```

Comment:
Checking the configure output shouldn't this depend on the mentioned libraries for the build (they are mentioned missing further along in the cmake configuration.

```
Building ...
cmake -H. -Bbuild -DCMAKE_BUILD_TYPE=Release -DWITH_AMALG=ON -DWITH_LUA_ENGINE=LuaJIT -DWithSharedLibluv=OFF -DWithOpenSSL=ON -DWithPCRE2=ON -DWithLPEG=ON -DWithZLIB=OFF -DWithOpenSSLASM=ON -DWithSharedOpenSSL=OFF -DWithSharedPCRE2=OFF -DWithSharedLPEG=OFF -DWithSharedZLIB=OFF
```